app/components/post_draft/draft_input/index.js

```diff
@@ -47,6 +47,7 @@ export function mapStateToProps(state, ownProps) {
             state,
             {
                 channel: channel.id,
+                team: channel.team_id,
```

Comment:
this permission check needs a teamId or else if the permission exists _only_ the team admin role for the user we wont check it ... although this was always returning true before anyways cause of the other issue with roleFound